File: Linguagem Java/retangulo.java
import java.util.Locale;
import java.util.Scanner;


public class retangulo {
	
	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Base do Retangulo: ");
		double base = sc.nextDouble();
		System.out.print("Altura do Retangulo: ");
		double altura = sc.nextDouble();
		double area = base * altura;
		System.out.println("Area = "+ String.format("%.4f", area));
		double perimetro = 2*base + 2*altura;
		System.out.println("Perimetro = "+ String.format("%.4f", perimetro));
		double diagonal = Math.sqrt(base*base + altura*altura);
		System.out.println("Diagonal = " + String.format("%.4f", diagonal));
		
		sc.close();
		
	}

}

File: Linguagem C++/retangulo.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {


    double base, altura, area, perimetro, diagonal;
    cout << "Base do retangulo: ";
    cin >> base;

    cout << "Altura do retangulo: ";
    cin >> altura;

    cout << fixed << setprecision(4);

    area = base * altura;
    cout << "Area = " << area << endl;

    perimetro = 2 * (base + altura);
    cout << "Perimetro = " << perimetro << endl;

    diagonal = sqrt(pow(base, 2) + pow(altura, 2));
    cout << "Diagonal = " << diagonal << endl;







    return 0;
}
